ipykernel/zmqshell.py

```diff
@@ -378,7 +378,7 @@ def connect_info(self, arg_s):
         print ("Paste the above JSON into a file, and connect with:\n"
             "    $> jupyter <app> --existing <file>\n"
             "or, if you are local, you can connect with just:\n"
-            "    $> jupyter <app> --existing {0}\n"
+            "    $> jupyter <app> --existing {}\n"
```

Comment:
Maybe replace `.format(connection_file)` with:
```suggestion
            f"    $> jupyter <app> --existing {connection_file}\n"
```